sources/ru/ranobelib.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import logging
+import json
+import operator
+
+from lncrawl.core.crawler import Crawler
+
+logger = logging.getLogger(__name__)
+
+
+class RanobeLibMeCrawler(Crawler):
+    base_url = [
+        "https://ranobelib.me/",
+    ]
+
+    def initialize(self):
+        self.init_executor(1)
+
+    def read_novel_info(self):
+        clean_url = self.novel_url.split("?")[0].strip("/")
+        logger.debug("Visiting %s", self.novel_url)
+        soup = self.get_soup(f"{clean_url}?section=info")
+
+        for script in soup.find_all("script"):
+            json_var = "window.__DATA__ = "
+            text = script.text.strip()
+            if not text or not text.startswith(json_var):
+                continue
+            text = text[len(json_var) : text.find("window._SITE_COLOR_")].strip(
+                ";\t\n "
+            )
+            content = json.loads(text)
+
+        self.novel_title = content["manga"]["rusName"]
+        logger.info("Novel title: %s", self.novel_title)
+
+        self.novel_cover = self.absolute_url(
+            soup.select_one(".media-sidebar__cover > img:nth-child(1)")["src"]
+        )
+        logger.info("Novel cover: %s", self.novel_cover)
+
+        self.novel_author = soup.select_one(
+            "div.media-info-list__item:nth-child(6) > div:nth-child(2) > a:nth-child(1)"
```

Comment:
handle the case where this tag might not exists. it should not throw error the crawler if novel author tag is not found. the author is optional.